README.md

```diff
@@ -1,20 +1,82 @@
 openshift-login
 ===============
 
-A Jenkins plugin which lets you login to Jenkins with your account on your OpenShift installation. Focuses on enabling SSO for Jenkins within your cluster.
+A Jenkins plugin which lets you login to Jenkins with your account on an OpenShift installation.
 
 
-The primary scenario is using this plugin when Jenkins is running in a OpenShift pod, and the jobs with Jenkins operate against the
-same OpenShift cluster that Jenkins is running in.  In this scenario, no additional configuration is required, but you must be running agaist v1.4 of OpenShift/Origin (https://github.com/openshift/origin/tree/release-1.4).
+Primary scenario
+--------------------------------
 
-Otherwise, for development purposes, or for scenarios where the OpenShift pod defaults do not apply, the configuration parameters are:
+This plugin can function with no additional configuration within Jenkins, but you must be running in an OpenShift Pod and against v1.4 of OpenShift/Origin (https://github.com/openshift/origin/tree/release-1.4).
 
-* service account directory:  The directory to load service account information from. Three files are referenced:  'namespace', 'ca.crt', and 'token'. They correspond to the OpenShift project, certificate, and authentication token for the service account.
-* service account name:  override for the service account name used when authenticating users against OAuth (default derived from token / client secret)
-* server prefix:  URI for the OpenShift OAuth endpoint
-* redirect URL: URL for the OpenShift API server
-* client ID:  override for the ID for the OpenShift OAuth client (default derived from service account information)
-* client secret:  override for the service account token (to change permissions for the OAuth client during the OAuth authentication flows)
+When running against that level of OpenShift/origin, and the plugin installed in your Jenkins instance, the authentication mechanism (the "Security Realm") established within your Jenkins instance is as follows:
+
+* If running outside of an OpenShift Pod, then on start up default Jenkins authentication is used, irregardless of the setting of the environment variable `OPENSHIFT_ENABLE_OAUTH`.  If you are using the [OpenShift Jenkins image](https://github.com/openshift/jenkins), you log in with the user name `admin`, supplying the password specified by the `JENKINS_PASSWORD` environment variable set on the container. If you do not override `JENKINS_PASSWORD`, the default password for `admin` is `password`.  For any other pre-existing Jenkins installation, your credentials into that specific system are needed. 
```

Comment:
ok, in that case the statement is still a bit misleading then.  I know we're trying to avoid the gorpy details about configuring the plugin manually, but someone who reads that sentence may not even make it down to the gorpy details below, or be confused about what to do with the OPENSHIFT_ENABLE_OAUTH variable.

So i think it needs to be clarified either here, or in the secondary scenario section below.